tutorials/community-start/community-start.md

```diff
@@ -176,7 +176,7 @@ To become a blogger in the SAP Community, you must follow a process that will he
 
 [ACCORDION-BEGIN [Step 10: ](Learn how to publish a blog post)]
 
-The more you know about writing a blog post in the community, the faster your first blog post will be processed.
+The more you know about writing a blog post in the community, the faster your first blog post will be processed. (Note: After submitting a post, a Contributor may need to wait up to 72 business hours before moderator review occurs. Contributors should wait for their first post to be published before submitting a second.)
```

Comment:
Can you use the standard Note coding (>) to enter the note? Skip a line and then add the note. Do not write "NOTE".

Reply:
I have updated as requested. Please let me know if I need to do anything else.